app_dart/test/request_handlers/check_for_waiting_pull_requests_test.dart

```diff
@@ -206,7 +206,8 @@ void main() {
       );
     });
 
-    test('Merges first PR in list, all successful', () async {
+    test('Merges first 2 PRs in list, all successful', () async {
```

Comment:
Can we have a test for 3 PRs, where either the first or second are not merged, but it still process all 3 PRs?

Reply:
Yes. For example, the oldest PR has failing checks so we don't merge it. However, the next oldest can be merged. I want to make sure that we don't break this functionality in the future.

Reply:
Done.